### PR TITLE
Feature1 implementation

### PR DESCRIPTION
**Pull request by robinshine**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -60,6 +60,8 @@ public interface DiffUtilsTest extends AppLoaderMocker {
 		assertEquals(3, diffBlocks.get(4).getNewStart());
 		assertEquals(3, diffBlocks.get(5).getOldStart());
 		assertEquals(4, diffBlocks.get(5).getNewStart());
+		assertEquals(5, diffBlocks.get(6).getNewStart());
```
Comment (robinshine) [inline test.js]: Please change 5 to 6 here.

File: test.js [hunk 2]
```diff
@@ -60,6 +60,8 @@ public interface DiffUtilsTest extends AppLoaderMocker {
 		assertEquals(3, diffBlocks.get(4).getNewStart());
 		assertEquals(3, diffBlocks.get(5).getOldStart());
 		assertEquals(4, diffBlocks.get(5).getNewStart());
+		assertEquals(5, diffBlocks.get(6).getNewStart());
+		assertEquals(6, diffBlocks.get(6).getNewStart());
```
Comment (robinshine) [inline test.js]: Should change getNewStart as getOldStart

File: file [hunk 3]
```diff
@@ -1,3 +1,3 @@
-dev1
+feature1
```
Comment (robinshine) [inline file]: Capitialize this sentence.

**Comment by yjshen324** [comment]
Is this what I want?

**Comment by robinshine** [comment]
May I help you sir?

File: file [hunk 4]
```diff
@@ -4,3 +4,4 @@ master branch
 Should we be prepared to add feature2
 Yes I agree
 Still be quiet
+This is great
```
Comment (yjshen324) [inline file]: great or not

File: file [hunk 5]
```diff
@@ -1,3 +1,7 @@
-dev1
+Feature1
 master
 master branch
+Should we be prepared to add feature2
```
Comment (robinshine) [inline file]: hello world

File: file [hunk 6]
```diff
@@ -1,3 +1,7 @@
...
+Yes I agree
+Still be quiet
```
Comment (robinshine) [inline file]: hello world

**Pull request merged**